src/JsonLd/ContextBuilder.php

```diff
@@ -97,6 +97,7 @@ public function getResourceContext(string $resourceClass, int $referenceType = U
             }
 
             $convertedName = $this->nameConverter ? $this->nameConverter->normalize($propertyName) : $propertyName;
+            $jsonldContext = $propertyMetadata->getAttributes()['jsonld_context'] ?? [];
```

Comment:
See for example how `hydra_context` is handled in https://github.com/api-platform/core/blob/7ac18660fc04f9c261238d447ac52993ff8786da/src/Hydra/Serializer/DocumentationNormalizer.php#L209-L273